### PR TITLE
feat(evals): SQL Evaluations

### PR DESCRIPTION
**Pull request by jlopatec**
Updated the SQL and Code functionality Evals to be non-experimental - in the package location

### CODE REVIEW

**Comment by mikeldking** [comment]
@anticorrelator can you take a look?

File: tutorials/evals/evals_sql_correctness_eval_with_custom_agent.ipynb [hunk 1]
```diff
@@ -0,0 +1,2574 @@
+{
```
Comment (anticorrelator) [inline tutorials/evals/evals_sql_correctness_eval_with_custom_agent.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #11.    from tqdm import tqdm
</pre><p>I think update your environment so your username isn't exposed here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2802/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (anticorrelator) [inline tutorials/evals/evals_sql_correctness_eval_with_custom_agent.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    !curl "https://www.sqlitetutorial.net/wp-content/uploads/2018/03/chinook.zip" -O ./chinook.zip
</pre><p>We can avoid saving a file to disk by piping the output of curl to unzip instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2802/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

File: tutorials/evals/evaluate_code_functionality_classifications.ipynb [hunk 2]
```diff
@@ -0,0 +1,2511 @@
+{
```
Comment (anticorrelator) [inline tutorials/evals/evaluate_code_functionality_classifications.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    import os
</pre><p>same here: update your environment so your username isn't exposed in this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2802/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Review by anticorrelator — COMMENTED** [review]
Thanks, this looks good! We just need to clean up the notebooks a little and then I think they're good to merge

**Comment by anticorrelator** [comment]
Closing in preference of #2861

**Pull request closed without merging**